### ISSUE TITLE
Auto-discover services not working?

### ISSUE THREAD

**Issue by erikvullings** [DRIVER-EU/test-bed-admin#8]
I can run the admin tool just fine, but it does not seem to pick up all services that are running? Do you automatically discover services, or are they predefined.

**Comment by tobritzh** [comment]
The admin tool shows alle the solutions, topics, gateways that are defined in the config files. The admin tool shows the state of all that preconfigured solutions/topics/gateways. No auto discover mode available.

**Comment by JoaquinMB-Freq** [comment]
then it's not a bug, right?

**Comment by erikvullings** [comment]
Yes, I saw the config files too late - however, do we really need them. At least it would be nice if the admin tool could create an overview of the available services based on the adapters it sees. And ask the user to which column an adapter (ID) belongs too. This information can be saved to the config file...
